### DISCUSSION TITLE
Unable to display images in portfolio posts

### DISCUSSION THREAD

**Discussion by ZimmZone** [earthlab-education/Earth-Analytics-AY24#650] [Q&A]
Hi Everyone, 

For some reason when I create a .html of my redlining project my image links are broken. I have tried adding them in by generating them with code, using .html, and even doing a screenshot and trying to add that to the file. 

To clarify, it shows up perfectly in the I notebook I generate in VS code as well as the  initial .html. But once I move that .html/notebook into my repository it longer works.

I think the issue is maybe something to do with where the files are being saved but I have tried a wide range of variations and have not been able to make it work. For instance, in the image below you can see that I have saved HTMLs in the images folder as well as the notebook folder. Can anyone point me in the right direction?

<img width="1190" alt="image" src="https://github.com/user-attachments/assets/047bcf24-cbee-4377-b761-efb770d59640">

**Comment by cmq879** [comment]
Hi, it would help to see the code where you call the image file to display it. 
Is it something like:
<embed type="text/html" src="img/NDVI_redlining.jpg" >
?

**Comment by nquarder** [comment]
Hey @ZimmZone I'm looking around in your portfolio repo to see if I can find the bug. One thing I'm noticing is with your [Arctic Tern post](https://zimmzone.github.io/Notebooks/arctic-tern-migration-final.html), which works great, it looks like at the bottom where you add the link to your interactive Arctic tern migration map you use the following html code:

`<embed height="1000" src="Arctic-tern-migration.html" type="text/html" width="1000"/>`

Here, you are linking back to the `Arctic-tern-migration.html` interactive map that is in the same `Notebooks` directory as your `arctic-tern-migration-final.html` notebook.

However, for the [redlining in Fort Collins post](https://zimmzone.github.io/Notebooks/fort_collins_portfolio_post2.html), it looks like you're linking to an image that possibly lives on your local machine based on what I'm seeing inside the `src=""` parameter below:

`<p>with open("images/my_image.png", "rb") as image_file:
encoded_string = base64.b64encode(image_file.read()).decode("utf-8")
img_tag = f'<img alt="Description" src="/Users/erinzimmerman/miniconda3/earth data analytics/fundamentals-04-redlining-ZimmZone/notebooks/images/NDVI_redlining.jpg" width="800"/>'
print(img_tag)</p>`

I would suggest adding the image of Fort Collins that you're trying to link to, to your `Notebooks` directory in your portfolio repo, where your `fort_collins_portfolio_post2.html` lives. Once they are in the same `Notebooks` directory you *should be able to do essentially what you did for the Arctic tern post, and you should see your image displaying at the bottom of your Fort Collins redlining post. Give this a shot and let us know how it shakes out. -NQ

**Reply by ZimmZone** [reply]
That worked! Thanks so much for looking into it!

**Reply by nquarder** [reply]
Great news @ZimmZone !
